devops/dev/config/logback-spring.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<configuration scan="true" debug="true" scanPeriod="15 seconds">
+  <include resource="logback/logback-spring-base.xml"/>
+
+  <springProfile name="!dev">
+    <root level="INFO">
+      <appender-ref ref="CONSOLE"/>
+    </root>
+  </springProfile>
+
+  <springProfile name="dev">
```

Comment:
Consider updating this so that we dont log in JSON by default in our local dev enviroment.

Reply:
Nice catch! 🙇